package.json

```diff
@@ -20,7 +20,8 @@
     "dev": "node ./scripts/esbuild.js",
     "start": "yarn dev",
     "test": "web-test-runner --coverage --config test/web-test-runner.config.js",
-    "publish-patch": "yarn run build && np patch --no-tests --any-branch"
+    "publish-patch": "yarn run build && np patch --no-tests --any-branch",
+    "publish-canary": "yarn run build && np $(semver $(npm pkg get version | sed 's/\"//g') -i prerelease --preid canary)+$(git rev-parse --short HEAD) --tag canary --no-release-draft --any-branch"
```

Comment:
I have no idea if this will work 😆 has this been tested yet? Not seeing any canary release cannel on npmjs, so probably only one way to know for sure which is to merge and let it run.

I'm guessing you've run the components of these commands to verify the output?

```
$(npm pkg get version | sed 's/\"//g')
```

```
$(git rev-parse --short HEAD)
```

```
semver {version} --tag canary --no-release-draft --any-branch
```

Reply:
npm dist tags can be created on the fly.

Reply:
yup all tested locally:
- npm pkg get version => "0.6.8"
- git rev-parse --short HEAD => c682c73
- semver is the node module NPM uses for versions, used to bump the version without editing package.json
- np is still used to publish without GH release notes